src/components/Label.vue

```diff
@@ -9,7 +9,7 @@ defineProps({
   <h4
     class="eyebrow border-b py-2 px-4 text-skin-text"
     :class="{
-      'sticky top-[72px] bg-skin-bg': sticky
+      'sticky top-[71.5px] bg-skin-bg': sticky
```

Comment:
This seem to have the same effect than using 71px directly, I can see on desktop that the "Voting" label will move slightly on Settings page when scrolling. Can't this be solve by doing 72px on large screen and 71px on small?

Reply:
Pushed commit to do that, but you will still have that slight jump on mobile still.

It's a issue in Chrome on HiDPI screens, alternative would be to implement it with JS to switch position automatically, way less elegant but shouldn't have those issues.